<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>playerperformance</class>
 <widget class="QWidget" name="playerperformance">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="area_2" native="true">
   <property name="geometry">
    <rect>
     <x>-20</x>
     <y>-10</y>
     <width>621</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, 
                                x1:0, y1:0, x2:0, y2:1, 
                                stop:0 rgba(6, 6, 6, 255), 
                                stop:0.41 rgba(56, 50, 52, 255), 
                                stop:1 rgba(34, 33, 33, 255));;
</string>
   </property>
   <widget class="QLabel" name="labelFirstName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>Player Performance</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditGoals">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAssists">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSaves">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>420</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color:qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  
border-radius:10px;
color:white;
}
QPushButton:hover {
    background-color:qlineargradient(x1: 0.25, y1: 0, x2: 0.75, y2: 1, 
                            stop: 0 rgba(0, 255, 254, 255),     
                            stop: 0.26 rgba(38, 170, 196, 255), 
                            stop: 0.67 rgba(89, 85, 131, 255),
                            stop: 1 rgba(0, 238, 255, 255));  
}</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelImage">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>12</y>
      <width>321</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Azonix</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color:yellow;
</string>
    </property>
    <property name="text">
     <string>label</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditYellows">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditReds">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>310</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="labelPlayTime">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>360</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
    background-color: white;
    padding: 5px;
    color: black;
    font-size: 16px;
}

/* Hover effect */
QLineEdit:hover {
	border:3px dashed qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}

/* Focus effect */
QLineEdit:focus {
    border:2px solid  qlineargradient(spread:pad, 
                            x1:0, y1:0, x2:1, y2:1, 
                            stop:0 rgba(255, 0, 0, 255), 
                            stop:0.41 rgba(195, 22, 58, 255), 
                            stop:1 rgba(255, 117, 0, 255));;  ;  ;
}
</string>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
